### PR TITLE
Prepares for release v2020.2 (Backend)

### PR DESCRIPTION
**Pull request by matsduf**

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,29 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v6.1.0 2020-11-06 (public release version)
+
+ [Features]
+ - Adds full support of a light weight SQLite database backend
+   as an alternative to MariaDB or PostgreSQL database daemons
+   (#695, #690, #177
+ - Extends zmtest and zmb with new functions (#675)
```
Comment (mattias-p) [inline Changes:9]: Maybe say something about the character of the new functionality?

File: Makefile.PL [hunk 2]
```diff
@@ -31,7 +31,7 @@ requires
   'Starman'                     => 0,
   'String::ShellQuote'          => 0,
   'Try::Tiny'                   => 0.24,
-  'Zonemaster::Engine'          => 4.0,
+  'Zonemaster::Engine'          => 4.001,
```
Comment (mattias-p) [inline Makefile.PL:34]: Does CLI make use of the new features introduced in Engine? Otherwise it's better not raise the version requirement.

**Review by mattias-p — APPROVED** [review]

File: Makefile.PL (hunk 2 above)
Comment (matsduf) [reply Makefile.PL:34]: You mean Backend.

If someone updates Backend by "cpanm Zonemaster::Backend" we want that to pull updated Zonemaster-LDNS and Zonemster-Engine too.

I have followed the instruction "[Update Makefile.PL with required version]".

We have had this discussion before. We have tested CLI with the updated Engine, v4.1.0, not with v4.0.0.

[Update Makefile.PL with required version]: https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/ReleaseProcess-release.md#5-update-makefilepl-with-required-version

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes]: Updated description.

File: Changes [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+v6.1.0 2020-11-06 (public release version)
+
+ [Features]
+ - Adds full support of a light weight SQLite database backend
+   as an alternative to MariaDB or PostgreSQL database daemons
+   (#695, #690, #177
+ - Extends zmtest with support for selecting profile. Extends zmb
+   with support for methods "profile_names" and "get_language_tags"
+   and support for selecting profile (#675)
+ - Installs zmtest and zmb test tools in path (#693, #687
```
Comment (pnax) [inline Changes:12]: Missing closing bracket.

File: Changes [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+ - Deprecates "force_hash_id_use_in_API_starting_from_id"
+   configuration item (#694)
+
+ [Fixes]
+ - Creates a work-around to allow for empty "ip" in nameserver
+   objects in "start_domain_test" method (#705, #702
```
Comment (pnax) [inline Changes:18]: Missing closing bracket.

File: Changes [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+ - Updates installation document (#701, #700, #699, #698, #697,
+   (#696, #695, #686
```
Comment (pnax) [inline Changes:20]: Missing closing bracket.
Too many opening brackets.

File: Changes [hunk 6]
```diff
@@ -1,5 +1,31 @@
...
+ - Removes unused configuration properties (#688, #273)
+ - Updates the configuration file document (#684, #520
```
Comment (pnax) [inline Changes:22]: Missing closing bracket.

File: Changes [hunk 7]
```diff
@@ -1,5 +1,31 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v6.1.0 2020-11-06 (public release version)
+
+ [Features]
+ - Adds full support of a light weight SQLite database backend
+   as an alternative to MariaDB or PostgreSQL database daemons
+   (#695, #690, #177
```
Comment (pnax) [inline Changes:8]: Missing closing bracket.

Comment (matsduf) [reply Changes]: Corrected all cases of missing parentheses.

**Review by pnax — APPROVED** [review]

**Review by mattias-p — APPROVED** [review]

**Pull request merged**